core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -459,7 +464,9 @@ var ExplorationEditorMainTab = function() {
   };
 
   this.expectTickMarkIsDisplayed = async function() {
-    expect(await correctAnswerTickMark.isDisplayed()).toBe(true);
+    await waitFor.visibilityOf(
+      correctAnswerTickMark, 'Correct answer tick mark not visible');
+    expect(correctAnswerTickMark.isPresent()).toBeTruthy();
```

Comment:
Can an element ever be visible but not present? If not (which is what I suspect), then the `expect(...)` here seems redundant

Reply:
I agree. Deleted it!